bitnami/mariadb-galera/values.schema.json

```diff
@@ -0,0 +1,193 @@
+          "description": "Name of the custom user to be created during the 1st initialization of MariaDB. This user only has permissions on the MariaDB custom database",
+          "form": true
+        },
+        "password": {
+          "type": "string",
+          "title": "Password for MariaDB custom user",
+          "form": true,
+          "description": "Defaults to a random 10-character alphanumeric string if not set",
+          "hidden": {
+            "condition": true,
+            "value": "existingSecret"
+          }
+        }
+      }
+    },
+    "ldap": {
+      "type": "object",
+      "title": "LDAP configuration",
+      "form": true,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Enable LDAP support",
+          "description": "Enable authentication using LDAP"
+        },
+        "uri": {
+          "type": "string",
+          "title": "LDAP URL",
+          "form": true,
+          "hidden": {
+            "condition": false,
+            "value": "ldap/enabled"
+          }
+        },
+        "base": {
+          "type": "string",
+          "title": "LDAP base DN",
+          "form": true,
+          "hidden": {
+            "condition": false,
+            "value": "ldap/enabled"
+          }
+        },
+        "binddn": {
+          "type": "string",
+          "title": "LDAP bind DN",
+          "form": true,
+          "hidden": {
+            "condition": false,
+            "value": "ldap/enabled"
+          }
+        },
+        "bindpw": {
+          "type": "string",
+          "title": "LDAP bind password",
+          "form": true,
+          "hidden": {
+            "condition": false,
+            "value": "ldap/enabled"
+          }
+        }
+      }
+    },
+    "replicaCount": {
+      "type": "integer",
+      "form": true,
+      "title": "Desired number of cluster nodes"
+    },
+    "persistence": {
+      "type": "object",
+      "title": "Persistence",
+      "form": true,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Enable persistence",
+          "description": "Enable persistence using Persistent Volume Claims"
+        },
+        "size": {
+          "type": "string",
+          "title": "Persistent Volume Size",
+          "form": true,
+          "render": "slider",
+          "sliderMin": 1,
+          "sliderMax": 100,
+          "sliderUnit": "Gi",
+          "hidden": {
+            "condition": false,
+            "value": "persistence/enabled"
+          }
+        }
+      }
+    },
+    "rbac": {
+      "type": "object",
+      "form": true,
+      "properties": {
+        "create": {
+          "type": "boolean",
+          "form": true,
+          "title": "Create RBAC rules",
+          "description": "Specify whether RBAC resources should be created and used"
+        }
+      }
+    },
```

Comment:
is this worth a parameter? I don't think disabling RBAC is a common use case

Reply:
it's disabled by default. This parameter allows you enabling it

Reply:
mmh, by default this chart doesn't work with a cluster with RBAC enabled?